### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by melissamcewen** [jdhines/gather-phase-4#3]
One thing I would have liked to see more is some comments, especially in the areas where it looks like you commented out some code? I wasn't sure why. Did the original block not work well? If so, why? 

https://github.com/jdhines/gather-phase-4/blob/master/test/features/user-visits-create-test.js#L15-L18

Phantom is a bit tricky, so it's totally understandable you might have to try some things

**Comment by jdhines** [comment]
Thanks! Wording is big for me as well; being that this was a tutorial, I didn't think about it as much as I should have. Fixed in 29af7ec

**Comment by jdhines** [comment]
Yes, that should have just been deleted. In that file specifically, the code was commented out because (and this was noted in Slack) the test would not pass with that code, which was the starter code. The test errored out with `Expected [Array(5)] to contain <test_title>`, I think because it was finding 5 images on the page, and `assert.include` doesn't do a deep look.
